grida-server/src/main/java/fr/insalyon/creatis/grida/server/business/PoolBusiness.java

```diff
@@ -93,13 +94,6 @@ public String addOperation(String proxyFileName, String source, String dest,
             Operation op = new Operation(id, source, dest, type, user,
                     proxyFileName, size);
             
-            long freeSpace = new File(".").getFreeSpace();
-            long totalSpace = new File(".").getTotalSpace();
-            if (freeSpace - size < totalSpace * Configuration.getInstance().getMinAvailableDiskSpace()) {
-                op.setStatus(Operation.Status.Failed);
-                logger.error("Unable to download '" + source + "' due to disk space limits. Size: " + ((int) size / 1024 / 1024) + " MB.");
-            }
-            
```

Comment:
we still want to verify here, that the download will be ok :
- already present 
- or enough space

Exactly as it's done in OperationBusiness::downloadSize